packages/docs/docs/guides/7-protocol/0-intro.md

```diff
@@ -1,10 +1,249 @@
 ---
-title: How the protocol works
-keywords: [Request protocol, IPFS, Ethereum, batching]
+title: Introduction to the Request Protocol
+keywords: [Request protocol, IPFS, Ethereum]
 
 ---
 
+Request is an open and unique database for payment requests including invoices or individual payment requests. It is aimed to be universal and to power products used by different companies from startups to large organizations, from the private to public sector.
+
+The Request Protocol is the core of Request. It's the bottom layer that defines and handles the data of a request and persist them to a distributed ledger to make Request open, trustless, secure and resilient.
+
+This document is aimed to help you to understand how the protocol is structured, how it works and how it meets its requirements.
+
 # Overview
 
-TODO by Lucas
+The Request Protocol has one basic purpose: **to persist, on a distributed ledger, data representing requests and to be able to retrieve these data in an efficient way**.
+
+To organize these different purposes, the Request Protocol follows the layered architecture pattern. Each layer is responsible for a specific task and a specific level of abstraction. This layered architecture also simplifies the understandability of the code, we believe it's an important matter for an open-source project.
+
+The protocol is composed of four layers:
+- Request logic
```

Comment:
missing  ̀advanced logic`

Reply:
As I already stated on Notion, advanced logic is not a layer in the Layered Architecture Pattern point of viewx